addons/stock/tests/test_inventory.py

```diff
@@ -311,6 +311,20 @@ def test_inventory_7(self):
         self.assertEqual(len(inventory.line_ids), 1)
         self.assertEqual(inventory.line_ids.theoretical_qty, 2)
 
+    def test_inventory_8(self):
+        """ Check inventory lines product quantity is 0 when inventory is
+        started with `prefill_counted_quantity` disable.
+        """
+        inventory_form = Form(self.env['stock.inventory'].with_context(
+                default_prefill_counted_quantity=False,
+             ), view='stock.view_inventory_form')
+        inventory = inventory_form.save()
+        inventory.action_start()
```

Comment:
maybe you can assert that some lines exist before checking their product_qty/theoretical_qty?
self.assertNotEqual(len(inventory.line_ids), 0)

Reply:
Yeah, that would be safer to do this check before :+1: